From e4eb9a139ba7d79855395d0c62cb3b5a851c668d Mon Sep 17 00:00:00 2001
From: TheGreatMcPain <james@thegreatmcpain.xyz>
Date: Mon, 5 Jun 2023 15:22:10 -0500
Subject: [PATCH] Tweaks for Gentoo ebuild

* xtask: Disable rpath in ffmpeg
* xtask: Don't build shared ffmpeg
---
 alvr/xtask/src/dependencies.rs | 2 --
 1 file changed, 2 deletions(-)

diff --git a/alvr/xtask/src/dependencies.rs b/alvr/xtask/src/dependencies.rs
index 621e206f..20a78c74 100644
--- a/alvr/xtask/src/dependencies.rs
+++ b/alvr/xtask/src/dependencies.rs
@@ -117,7 +117,6 @@ pub fn build_ffmpeg_linux(nvenc_flag: bool) {
         "--enable-gpl",
         "--enable-version3",
         "--enable-static",
-        "--enable-shared",
         "--disable-programs",
         "--disable-doc",
         "--disable-avdevice",
@@ -136,7 +135,6 @@ pub fn build_ffmpeg_linux(nvenc_flag: bool) {
         "--enable-vulkan",
         "--enable-libdrm",
         "--enable-pic",
-        "--enable-rpath",
     ];
     let install_prefix = format!("--prefix={}", final_path.join("alvr_build").display());
     // The reason for 4x$ in LDSOFLAGS var refer to https://stackoverflow.com/a/71429999
-- 
2.39.3

